libmscore/rest.cpp

```diff
@@ -90,29 +102,32 @@ void Rest::draw(QPainter* painter) const
             //only on voice 1
             if (track() % VOICES)
                   return;
-            Measure* m = measure();
-            int n      = m->mmRestCount();
+
+            // draw number
+            int n      = measure()->mmRestCount();
```

Comment:
very strange indentation

Reply:
The previous use justified it, but it shouldn't be needed anymore.

Reply:
Fixed